spec/services/distribution_reminder_spec.rb

```diff
@@ -12,40 +12,57 @@
 
     context 'when the distribution_id does not match any Distribution' do
       it "does not send mail for non existent distributions" do
+        job = double(Delayed::Backend::ActiveRecord::Job, reminder_email: nil)
+        allow(DistributionMailer).to receive(:delay).and_return(job)
         DistributionReminder.perform(0)
-        expect(DistributionMailer.method(:reminder_email)).not_to be_delayed
+
+        expect(job).not_to receive(:reminder_email)
```

Comment:
@m-pereira I've got it! So for each `receive` you'll need to use `have_received` instead. This is because `have_received` will pick up that the `job` has received when calling `DistributionReminder`

I think that would fix a majority of the issues :)!

Reply:
Nice, I'll try with `have_received` matcher

Reply:
Nice! Looks like it fixed all broken specs